### PR TITLE
QVR Pro documentation

### PR DESCRIPTION
**Pull request by oblogic7**
## Proposed change

Adds documentation for new QVR Pro integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant/pull/31173
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please merge the documentation to one single file `qvrpro.markdown` and put it in the `_integrations` folder.

**Comment by frenck** [comment]
It is not just the renaming, but the whole page.

It seems like you are using a very very old format (over a year old at least).
Please take a look at other pages, the way we make headers and things we put into the frontmatter changed quite a bit.

Please update this page to match the rest of our documentation.

**Comment by oblogic7** [comment]
Yes, I started this integration over a year ago and the working branch was rather stale.  I believe everything should be current now.

File: source/_integrations/qvr_pro.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Configuration
+
+To enable QVR Pro integration, add the following to your
+`configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+qvr_pro:
+  host: 192.168.1.102
```
Comment (frenck) [inline source/_integrations/qvr_pro.markdown]:
```suggestion
  host: YOUR_HOST
```

File: source/_integrations/qvr_pro.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  username: pyqvruser
```
Comment (frenck) [inline source/_integrations/qvr_pro.markdown]:
```suggestion
  username: YOUR_USERNAME
```

File: source/_integrations/qvr_pro.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  password: password
```
Comment (frenck) [inline source/_integrations/qvr_pro.markdown]:
```suggestion
  password: YOUR_PASSWORD
```

File: source/_integrations/qvr_pro.markdown [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  exclude_channels: 2,5,6
```
Comment (frenck) [inline source/_integrations/qvr_pro.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_integrations/qvr_pro.markdown [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+ha_category:
+  - Camera
+ha_release: "0.106"
+---
+
+[QVR Pro](https://www.qnap.com/solution/qvr-pro-official) allows you to create 
+an independent and expandable surveillance environment on your QNAP NAS.  The 
+`qvr_pro` component allows you to view your QVR Pro channels in Home Assistant.
+
+Currently only cameras are supported by this integration.
```
Comment (frenck) [inline source/_integrations/qvr_pro.markdown]:
```suggestion
Currently, only cameras are supported by this integration.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Pull request merged**